cvat/apps/lambda_manager/views.py

```diff
@@ -98,6 +102,8 @@ def __init__(self, gateway, data):
                 "`{}` lambda function has non-unique labels".format(self.id),
                 code=status.HTTP_404_NOT_FOUND)
         self.labels = labels
+        # mapping of labels and corresponding supported attributes
+        self.attributes = {item['name']: item.get('attributes', []) for item in spec}
```

Comment:
@mikhail-treskin , OK. The code looks good. Probably it can be a good idea to check that for the same label we don't have duplicated attributes (as in the code for labels above).